pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -54,19 +55,22 @@ var _ webhook.Validator = &Cluster{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *Cluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", r.Name)
-	if r.IsReconcilePaused() {
-		clusterlog.Info("cluster is paused, so allowing create", "name", r.Name)
-		return nil
-	}
-	if !features.IsActive(features.FullLifecycleAPI()) {
-		return apierrors.NewBadRequest("Creating new cluster on existing cluster is not supported")
+	var allErrs []error
+	if err := r.Validate(); err != nil {
+		allErrs = append(allErrs, fmt.Errorf("invalid cluster: %v ", err))
```

Comment:
Do we want to append all errors from the `Validate()` method? Because it seems that it will return after the first error & only append that one error to `allErrs`

Reply:
Yes we only append the first error from Validate. Appending everything from Validate would require changes in the Validate method itself, which I believe are not exactly in scope for this task. We could explore it though in the future.